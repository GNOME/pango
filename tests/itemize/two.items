# test that font_features attributes don't break the
# runs anymore (but other attributes do)
<span font="Cantarell 11">one <span font_features="tnum=0">tw<u>o</u> <span font_features="dlig=1">two</span> </span>th<b>r</b>ee</span>
