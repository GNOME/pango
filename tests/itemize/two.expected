<span font="Cantarell 11">one <span font_features="tnum=0">tw<u>o</u> <span font_features="dlig=1">two</span> </span>th<b>r</b>ee</span>

Items:  one tw                    |o                                          | two th                                              |r                |ee          
Font:   Cantarell 11              |Cantarell 11                               |Cantarell 11                                         |Cantarell Bold 11|Cantarell 11
Script: latin                     |latin                                      |latin                                                |latin            |latin       
Lang:   en-us                     |en-us                                      |en-us                                                |en-us            |en-us       
Bidi:   0                         |0                                          |0                                                    |0                |0           
Attrs:  [4,12]font-features=tnum=0|[6,7]underline=1,[4,12]font-features=tnum=0|[8,11]font-features=dlig=1,[4,12]font-features=tnum=0|                 |            
